xfields/beam_elements/beambeam3d.py

```diff
@@ -308,6 +379,12 @@ def __init__(self,
         self.min_sigma_diff = min_sigma_diff
         self.threshold_singular = threshold_singular
 
+        self.flag_beamstrahlung = flag_beamstrahlung
+        self.flag_luminosity = flag_luminosity
+        self.turn = turn,
+        #self.flat_y_i=flat_y_i,
```

Comment:
Remember to remove these

Reply:
see [22 seconds ago](https://github.com/xsuite/xfields/pull/50#discussion_r1006490301)

Reply:
removed